RetailSuiteAccounting/Overview/04_CountraAccountSpecialFunctions.md

```diff
@@ -0,0 +1,46 @@
+|-------------|-------------|-------------|
+| 0 / empty | without cancellation key | This cancellation key is used when no cancellation key has been specified. The meaning of the tax key is as follows: <br> &bull; 0: No tax posting <br> &bull; 1: No meaning <br> &bull; 2: Reduced VAT rate (7%) <br> &bull; 3: Standard VAT rate (19%) <br> &bull; 5: Old standard VAT rate (16%) <br> &bull; 7: Old standard deductible input tax rate (16%) <br> &bull; 8: Reduced deductible input tax rate (7%) <br> &bull; 9: Standard deductible input tax rate (19%)|
+| 1 | EU posting | The cancellation key 1 triggers, together with a tax key, a EU posting. In this case, the tax keys have the following meaning: <br> &bull; 0: Supply of goods/services taxable in another EU country <br> &bull; 1: Tax-free supply of goods within the EU <br> &bull; 2: Reduced VAT rate for domestic taxable supplies of goods within the EU (7%) <br> &bull; 3: Standard VAT rate for domestic taxable supplies of goods within the EU (19%) <br> &bull; 5: Old standard VAT rate for domestic taxable supplies of goods within the EU (16%) <br> &bull; 7: Old standard VAT rate for intra-community taxable acquisitions of goods (16%) <br> &bull; 8: Reduced VAT rate for intra-community taxable acquisitions of goods (7%) <br> &bull; 9: Standard VAT rate for intra-community taxable acquisitions of goods (19%)|
+| 2 | General reversal | The cancellation of a "normal" posting is performed by entering the number 2 in the first digit of the contra account. The turnover is then entered with the reversed sign. The resulting posting cancels the selected posting without deleting it.|
+| 3 | General reversal with distributable input tax | The cancellation key 3 is used to cancel postings that have been entered with the cancellation key 9.|
+| 4 | Cancellation of automatic function | By entering the cancellation key 4, the automatic deduction of input tax or VAT is blocked. Besides, collective accounts can also be posted to by entering the tax key 4.|
+| 5 | Individual tax key | The cancellation key 5 allows to enter multi-digit tax keys. For this purpose, a window opens immediately when leaving the *Contra account* field. |
+| 6 | General reversal with key 1 - EU-posting | The cancellation key 6 is used to cancel postings that have been entered with the cancellation key 1. In this case, the original tax key must be specified as well.|
+| 7 | General reversal with key 5 - Individual tax key | The cancellation key 7 is used to cancel postings that have been entered with the cancellation key 5.|
+| 8 | General reversal with key 4 - Cancellation of the posting automatic function | The cancellation key 8 is used to cancel postings that have been entered with the cancellation key 4.|
+| 9 | Distributable input tax | The cancellation key 9 triggers, together with the tax keys 6 to 9, a posting of the input tax to the accounts for distributable input tax (instead of to the accounts for deductible input tax).|
```

Comment:
```suggestion
| 9 | Distributable input tax | The cancellation key **9** triggers, together with the tax keys **6** to **9**, a posting of the input tax to the accounts for distributable input tax (instead of to the accounts for deductible input tax).|
```